addons/website_forum/models/forum.py

```diff
@@ -139,7 +139,7 @@ def _check_default_post_type(self):
         if (self.default_post_type == 'question' and not self.allow_question) \
                 or (self.default_post_type == 'discussion' and not self.allow_discussion) \
                 or (self.default_post_type == 'link' and not self.allow_link):
-            raise UserError(_('You cannot choose %s as default post since the forum does not allow it.') % self.default_post_type)
```

Comment:
Why removing the value import, this will display "You cannot choose %s as default post since the forum does not allow it."